Readme.md

```diff
@@ -76,6 +76,10 @@
 - [Audio Ghar](https://play.google.com/store/apps/details?id=com.audioghar.app)
 - [BookSuna:Audio Books](https://play.google.com/store/apps/details?id=com.ngp.booksuna)
 
+### Games
```

Comment:
Let's add this category in Contents. I am not good with markdown as you can see, I don't know how to auto-generate the contents section. 😀😀

Thank You for the PR.

Reply:
Sure, I have added the games section in contents.